app/db/models.py

```diff
@@ -1,54 +1,113 @@
+                primary_key=True, nullable=False, index=True)
+    # Which model does this grid belong to? e.g. GDPS latlon.15x.15?
+    prediction_model_id = Column(Integer, ForeignKey(
+        'prediction_models.id'), nullable=False)
+    prediction_model = relationship("PredictionModel")
+    # Order of vertices is important!
+    # 1st vertex top left, 2nd vertex top right, 3rd vertex bottom right, 4th vertex bottom left.
+    geom = Column(Geometry('POLYGON'), nullable=False)
+    predictions = relationship(
+        "Prediction")
+    # "Prediction", back_populates="prediction_model_grid_subset")
+
+
+class Prediction(Base):
+    """ The particular prediction.
+    Each value is an array that corresponds to the vertex in the forecast bounding polygon. """
+    __tablename__ = 'predictions'
+    __table_args__ = (
+        UniqueConstraint('prediction_run_id', 'prediction_model_grid_subset_id',
+                         'prediction_timestamp'),
+    )
 
-    id = Column(Integer, Sequence('single_forecast_id_seq'),
+    id = Column(Integer, Sequence('predictions_id_seq'),
                 primary_key=True, nullable=False, index=True)
-    """ The date and time to which the forecast applies """
-    forecast_datetime = Column(DateTime, nullable=False)
-    forecast_model = Column(Integer, ForeignKey(
-        'forecast_model.id'), nullable=False)
-    weather_station = Column(Integer, nullable=False)
-    """ The date and time at which the model was run / forecast was issued """
-    issue_date = Column(DateTime, nullable=False, default=func.now())
-    temperature = Column(Float)
-    dew_point = Column(Float)
-    relative_humidity = Column(Float)
-    wind_speed = Column(Float)
-    wind_direction = Column(Float)
-    total_precipitation = Column(Float)
-    accumulated_rain = Column(Float)
-    accumulated_snow = Column(Float)
-    accumulated_freezing_rain = Column(Float)
-    accumulated_ice_pellets = Column(Float)
-    cloud_cover = Column(Float)
-    sea_level_pressure = Column(Float)
-    wind_speed_40m = Column(Float)
-    wind_direction_40m = Column(Float)
-    wind_direction_80m = Column(Float)
-    wind_speed_120m = Column(Float)
-    wind_direction_120m = Column(Float)
-    wind_speed_925mb = Column(Float)
-    wind_direction_925mb = Column(Float)
-    wind_speed_850mb = Column(Float)
-    wind_direction_850mb = Column(Float)
+    # Which model run does this forecacst apply to? E.g. The GDPS 15x.15 run from 2020 07 07 12h00.
+    prediction_run_id = Column(Integer, ForeignKey(
+        'prediction_runs.id'), nullable=False)
+    prediction_run = relationship(
+        "PredictionRun", foreign_keys=[prediction_run_id])
+    # Which grid does this forecast apply to?
+    prediction_model_grid_subset_id = Column(Integer, ForeignKey(
+        'prediction_model_grid_subsets.id'), nullable=False)
+    # prediction_model_grid_subset = relationship(
+    # "PredictionModelGridSubset", back_populates="predictions")
```

Comment:
should this still be commented out? or deleted?

Reply:
My bad.